src/instructlab/model/evaluate.py

```diff
@@ -139,18 +139,33 @@ def validate_model(model: str, model_arg: str = "--model"):
         raise click.exceptions.Exit(1)
 
 
-def sort_score(pairing: tuple[str, float]) -> float:
+def sort_score(pairing: tuple[str, float, float, float]) -> float:
     """helper func for display_branch_eval_summary
     takes a tuple pairing and returns just the score
     """
     return pairing[1]
 
 
-def display_models(model, base_model) -> None:
+def get_benchmark_max_score(benchmark: Benchmark) -> str:
```

Comment:
These max score / score range changes should be made for the non _branch variants as well